libr/core/cmd_search.c

```diff
@@ -500,38 +487,35 @@ static int __cb_hit(RSearchKeyword *kw, void *user, ut64 addr) {
 			}
 		} else {
 			r_cons_printf ("0x%08"PFMT64x " %s%d_%d %s\n",
-				base_addr + addr, searchprefix, kw->kwidx, kw->count, s);
+				base_addr + addr, searchprefix, kw->kwidx, kw->count - 1, s);
```

Comment:
i would move the logic that creates a flagname for a keyword into a separate function and reuse it in all the places that it constructs the string. this will save us from inconsistencies, honor searchprefix always, avoid negative counts, etc

Reply:
I know, but not in this commit